pallets/ocex/src/types.rs

```diff
@@ -13,7 +13,7 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 // GNU General Public License for more details.
 
-use codec::{Decode, Encode, MaxEncodedLen};
+use parity_scale_codec::u{Decode, Encode, MaxEncodedLen};
```

Comment:
Fatfingering with `u`? Or global rename? :)

Reply:
this file is not in module tree... should I remove it @Gauthamastro ?

Reply:
please remove it